internal/server/resolvers/manifest_merge.go

```
@@ -0,0 +1,254 @@
+		Os:        "manifest",
+		Reachable: false,
+		Peers:     []*graphql1.Host{},
+	}
+	applyManifestEntry(h, entry, true)
+	return h
+}
+
+// hostKey returns the merge key for a Host. Prefers MachineID (used by
+// peerproxy + the host provider); falls back to Hostname so manifest-
+// only entries dedupe against live-fleet rows that share a hostname.
+func hostKey(h *graphql1.Host) string {
```

golangci-lint (2.12.2)

[error] func hostKey is unused

(unused)